client/src/auth/SignUp.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import { NavLink, Route } from 'react-router-dom';
+
+class SignUp extends Component {
+  state = {
+    username: 'Davo',
+    password: 'Test',
+    department: 'admin'
+  };
+
+  handleInputChange = event => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value
+    });
+  };
+
+  handleSubmit = async event => {
+    try {
+      event.preventDefault();
+      const endpoint = 'http://localhost:4000/api/register';
+      const res = await axios.post(endpoint, this.state);
+      localStorage.setItem('jwt', res.data.token);
+
+      console.log('Second endpoint!');
+      const loginEndpoint = 'http://localhost:4000/api/login';
+      const loginRes = await axios.post(loginEndpoint, this.state);
```

Comment:
Does this work w/ doing 2 different endpoints in the same action? It looks like it'd just be overwriting the previous endpoint call since they're being save to the same key in the local storage.